comments-by-guideline-and-success-criterion.md

```diff
@@ -566,22 +566,22 @@ See also the [Comments on Closed Functionality](#comments-on-closed-functionalit
 
 ###### Applying SC 2.4.2 Page Titled to Non-Web Documents and Software
```

Comment:
Per TF resolution on 15 May to use headings to distinguish between non-web document and non-web software guidance.
```suggestion
###### Applying SC 2.4.2 Page Titled to Non-Web Documents
```